### ISSUE TITLE
configure HTTP recording from the command line

### ISSUE THREAD

**Issue by Aarskin** [linkedin/flashback#14]
Hi, 

I'm hoping there is a way to configure my proxy with a trusted certificate from the command line. I don't see this anywhere in the current set of samples - only Java examples. 

Is there a way to configure a certificate for my Flashback proxy without Java?

Thanks

**Comment by Aarskin** [comment]
I was able to get 200 OK with

`{
  "sceneMode": "record",
  "sceneName": "test1",
  "matchRule": "matchEntireRequest",
  "scenePath": "tmp",
  "proxyHost": "localhost",
  "proxyPort": "5555",
  "caCertPath": "C:\\certs\\trust_proxies.pfx",
  "caCertPwd": "P@ssw0rd@1",
  "caAlias": "ITrustProxies",
  "caKeyPwd": "P@ssw0rd@1",
  "caCertCN": "",
  "caCertOU": "",
  "caCertO": "",
  "caCertL": "",
  "caCertCC": ""
}`

But I am still unable to hit pages with HTTPS using Flashback as a MITM. Is there any way I can verify that this is a trust issue with my Certificate and not with Flashback itself?

I used IIS to generate a self-signed certificate. I do not get a security alert in IE or Chrome, but I do see the broken lock (in Chrome, closed lock in IE). Can anyone confirm/deny that this is the root of problem?

**Comment by sf1152** [comment]
Yes, the way we support non-java is that we have admin restful api that can setup proxy for you. You just need make sure if your caCert can be trusted by your certificate store.

BTW: did you get any response from proxy?

**Comment by Aarskin** [comment]
Not quite yet, I'm focusing on getting more comfortable with other aspects of Flashback at the moment.

**Comment by yabin-kang** [comment]
@Aarskin hi, how did you try to access a https resource using flashback, curl? not sure about windows, but here is more details of how generate and using a self signed certificate for flashback here: https://github.com/linkedin/flashback/issues/8
